include/votca/tools/eigen.h

```diff
@@ -44,6 +44,7 @@
 #pragma GCC diagnostic ignored "-Wint-in-bool-context"
 #endif
 #include <Eigen/Eigen>
+#include <unsupported/Eigen/CXX11/Tensor>
```

Comment:
Is it really in a folder `unsupported/` ?

Reply:
yeah. It is an unofficial module of eigen but since google uses it in tensorflow. I think it is good enough

Reply:
ok

Reply:
https://stackoverflow.com/questions/41518379/why-was-eigen-chosen-for-tensorflow